packages/contracts/contracts/chugsplash/L2/ChugSplashDeployer.sol

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: MIT
+// @unsupported: evm
+pragma solidity >0.5.0 <0.8.0;
+pragma experimental ABIEncoderV2;
+
+/* Library Imports */
+import { Lib_ExecutionManagerWrapper } from "../../optimistic-ethereum/libraries/wrappers/Lib_ExecutionManagerWrapper.sol";
+import { Lib_MerkleTree } from "../../optimistic-ethereum/libraries/utils/Lib_MerkleTree.sol";
+
+/**
+ * @title ChugSplashDeployer
+ */
+contract ChugSplashDeployer {
```

Comment:
Btw one high level bit of feedback here is I think that more 'deployment'-esque words could make the operation of this contract clearer. For instance `hasActiveBundle` seems more implicit than `hasActiveDeployment`. 

I feel like it's OK to have deploy-specific terminology here because the `ChugSplashDeployer` is going to have to handle logic for things like shutting down the contracts to begin a deploy, and restarting them to end the deploy. So idk it seems like it's low value to make things more abstract than they need to be at least at first. Eventually we can break it out into a `BundleExecutor` or something like that, but in the meantime I just like reminding folks what they are reading